pkgs/applications/office/gnucash/default.nix

```diff
@@ -1,106 +1,210 @@
-{ fetchurl, lib, stdenv, pkg-config, makeWrapper, cmake, gtest
-, boost, icu, libxml2, libxslt, gettext, swig, isocodes, gtk3, glibcLocales
-, webkitgtk, dconf, hicolor-icon-theme, libofx, aqbanking, gwenhywfar, libdbi
-, libdbiDrivers, guile, perl, perlPackages
+{
+  # nix "native" dependencies
+  fetchurl
+, lib
+, stdenv
+
+, glibcLocales  # necessary to pass locale-related GnuCash tests
+, wrapGAppsHook # nix sugar for Gnome apps
+
+, makeWrapper   # used to wrap gnucash with Gnome sugar and the cli utils which need Perl
+
+, cmake         # tell nix to use cmake rather than make
+
+  # GnuCash requirements
+, aqbanking
+, boost
+, glib
+, gtest
+, guile
+, gwenhywfar
+, icu
+, libdbi
+, libdbiDrivers
+, libofx
+, libxml2
+, libxslt
+, perl
+, pkg-config
+, swig
+, webkitgtk
+
+  # GnuCash cli utils' Perl requirements
```

Comment:
```suggestion
```

Reply:
fixed in 072900f31f1598aca35feb66e4c553bbe69a9509